datafusion/sql/tests/cases/plan_to_sql.rs

```diff
@@ -1146,6 +1146,33 @@ fn test_join_with_table_scan_filters() -> Result<()> {
 
     assert_eq!(sql.to_string(), expected_sql);
 
+    let right_plan_with_filter_schema = table_scan_with_filters(
```

Comment:
These are tests for the SQL unparsing

Is there any chance you could add an EXPLAIN test in .slt instead?

Perhaps somewhere in datafusion/sqllogictest/test_files/push_down_filter.slt?

Reply:
The instructions for doing so are in https://github.com/apache/datafusion/blob/main/datafusion/sqllogictest/test_files/string/README.md